modules/lockrAIMIdSystem.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * This module adds lockr AIM ID support to the User ID module
+ * The {@link module:modules/userId} module is required.
+ * @module modules/lockrAIMIdSystem
+ * @requires module:modules/userId
+ */
+
+import { submodule } from "../src/hook.js";
+import { logInfo, logWarn } from "../src/utils.js";
+// eslint-disable-next-line prebid/validate-imports
+import { lockrAIMGetIds } from "./lockrAIMIdSystem_shared.js";
```

Comment:
Do you expect other modules to call these methods?

Reply:
No, we do not.

Reply:
Cool, Please don't make a cross-module library for methods only one module calls